testing/asserts.ts

```diff
@@ -864,3 +866,58 @@ export function unimplemented(msg?: string): never {
 export function unreachable(): never {
   throw new AssertionError("unreachable");
 }
+
+let snapshotFile: Record<string, string> | undefined = undefined;
+const updatedSnapshotFile: Record<string, unknown> = {};
+const snapshotMap: Record<string, number> = {};
+
+export async function assertSnapshot(context: Deno.TestContext, actual: unknown) {
+  const name = getName(context);
+  const count = getCount();
+  const testName = `${name} #${count}`;
```

Comment:
```suggestion
  const testName = `${name} ${count}`;
```

This makes it consistent with Jest snapshots, which don't have a #.

Reply:
Done